recipes/uutils/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "uutils" %}
```

Comment:
Shouldn't the name of this package be `uutils-coreutils`? We already have a `GNU-coreutils` package. This is a replacement for that package, right? If the have the same API/named outputs, then you also need to make these packages mutually exclusive using `run_constrained` so they don't squash each other's artifacts.

https://github.com/conda-forge/coreutils-feedstock
https://conda-forge.org/feedstock-outputs/

Reply:
Adding the `-coreutils` suffix would be a nice hint of clobbering artifacts. Thanks for the careful review!